qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -18,8 +18,9 @@
     backup_file = /etc/sysconfig/qemu-ga-bk
     black_list_backup = /bin/cp -f /etc/sysconfig/qemu-ga ${backup_file}
     recovery_black_list = mv -f ${backup_file} /etc/sysconfig/qemu-ga
-    #only for rhel8,virt module stream is rhel or 8.1,8.2,8.3 and so on
-    virt_module_stream = rhel
+    cmd_check_qga_installlog = "ls /qga_install_situation.log"
```

Comment:
Please update the file name.

Reply:
@yanan-fu, done.

Reply:
@6-dehan  Seems no change, could you double confirm ? thanks!

Reply:
@yanan-fu  sorry about that. something went wrong at that time. It's okay now.